src/components/scrollable-list/scrollable-list-item.component.js

```diff
@@ -0,0 +1,20 @@
+import styled, { css } from 'styled-components';
+import propTypes from 'prop-types';
+
+const StyledListItem = styled.li`
+  width: 100%;
+  padding: 5px 6px;
+  ${({ isSelected }) => isSelected && css`background-color: #E6E9E7;`}
+  height: 15px;
+`;
+
+StyledListItem.propTypes = {
+  id: propTypes.number,
+  isSelected: propTypes.bool
+};
+
+StyledListItem.defaultProps = {
+  isSelectable: true
```

Comment:
I would perhaps add a comment next to this one to describe why it is required, as it wasn't clear to me before I figured out the API you're using. eg. `// defaulted to true so it integrates with ScrollableList by default`

Reply:
The `css` object is used when you need to add  a whole line of css, ie. the attr and value. In this instance I do not give a `background-color` at all unless there is a prop, so I use the css object add the line. If you prefer I can just set the background to  `transparent` by default?

Reply:
I think it'll work without using it, but I found this thread explaining why you should use it regardless https://github.com/styled-components/styled-components/issues/1178#issuecomment-332211886

Reply:
I think thats true as they type, but not on open - see https://carbon.sage.com/components/dropdown-filter

It may be that the select/option components will dictate this. Leave it for now and we will see how it works when we plug it all together 👍